guidelines/terms/21/single-pointer.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn>single pointer</dfn></dt>
 <dd>
-   	
-   <p>pointer input that operates with one point of contact with the screen, including single taps and clicks, double-taps and clicks, long presses, and path-based gestures</p>
-  				
+   <p>an input that only targets a single point on the page/screen at a time – such as a mouse, stylus, or single finger on a touch screen – as opposed to multi-pointer/multi-touch scenarios</p>
```

Comment:
```suggestion
   <p>an input that only targets a single point on the page/screen at a time. For example a mouse, stylus, or single finger on a touch screen rather than multi-pointer/multi-touch input</p>
```

Reply:
```suggestion
   <p>an input that only targets a single point on the page/screen at a time – such as a mouse, stylus, or single finger on a touch screen</p>
```